### DISCUSSION TITLE
routing issue south of Hobro, DK

### DISCUSSION THREAD

**Discussion by MyHekla** [osmandapp/OsmAnd#21210] [General]
The issue seems to be around the roundabout on route 180 south of Hobro. The issue is only in tha APP(android). No difference whichever map source I chose.

**Comment by ERYpTION** [comment]
What issue specifically?

**Reply by MyHekla** [reply]
Make a starting point on route 180 just south of Hobro, DK - southbound through the next roundabout or to the next city on route 180 in OSMAND.

**Reply by ERYpTION** [reply]
Like so?
![Screenshot_20241106-191958](https://github.com/user-attachments/assets/5e2fc921-e60b-46b3-ae46-8327dfc07f78)

**Reply by MyHekla** [reply]
that's how it should be. But on the app it does some strange routing where 555 joins in the roundabout.

**Reply by ERYpTION** [reply]
Can't replicate in the app on my phone.

**Reply by MyHekla** [reply]
this is the way it looks on my phone and tablet
<table style="width:100%">
  <tr>
<th><img src="https://github.com/user-attachments/assets/99894bc4-0e43-4f16-aa6d-c252ec4c7ed3"  width="300" height="auto"/> </td>
  </tr>
</table>

**Reply by MyHekla** [reply]
well just updated to the November mas and now there is difference between car and truck navigation. Car correct, truck wrong.
So this might be a map issue.

**Reply by ERYpTION** [reply]
I see a 'hgv:conditional' and a 'maxweight:conditional' tag on the north entrance(s) to the roundabout. Valid between 07:00-18:00. Don't know if this is correct but probably the cause of the different routing for trucks.

https://www.openstreetmap.org/way/749258127

**Reply by MyHekla** [reply]
exactly - that is not correct. The limit is drive-through Hobro city. It is allowed to drive into Hobro - unload/load - then continue through.
